### ISSUE TITLE
Convert Kit wrapper

### ISSUE THREAD

**Issue by earthwitch** [ApothecaryGarden/mtollwc#9]
so we can:
- respond to cancellations
- have email changes convert across the board
- have autoresponders for badges

**Comment by oakwoodgates** [comment]
Let's get a list going of the different scenarios when would want to send or receive data from WC to CK. 

For example:
- If user edits info in /my-account/edit-account/, sync first name, last name, and email
- If user orders this subscription, do this
- If user cancels this subscription, do this

**Comment by earthwitch** [comment]
It's important for @maelamb to chime in here cause she does all the Convert Kit stuff!

My thoughts beyond what you have above:
- autoresponders to users and to admin when badges are acquired
- if payment fails on a subscription (ie they still want to be subscribed but need to update credit card info)

**Comment by oakwoodgates** [comment]
ok @earthwitch @maelamb do we want to start collaborating on a list in a google doc?

**Comment by maelamb** [comment]
Good idea on the google doc, @oakwoodgates. I'm setting one up now and will send the link in a moment. :)

**Comment by oakwoodgates** [comment]
@earthwitch @maelamb 
Let's keep this in mind too...
Version 2 of the API http://kb.convertkit.com/article/api-basics/
It allows:
- Get a list of forms for an account
- Subscribe to a form
- Subscribe to a sequence
- Add subscriber to an email sequence
- Mark a subscriber as converted
- Unsubscribe from a form
- Unsubscribe from a sequence

Version 3 of the API is in active development. http://kb.convertkit.com/article/api-documentation-v3/
Currently it allows you to:
- Get a list of forms for an account
- Add subscriber to a form
- Add subscriber to an email sequence
- Tag a subscriber
- Remove tags from a subscriber

**Comment by oakwoodgates** [comment]
I would like to and recommend to use v3 if possible.

**Comment by earthwitch** [comment]
How can we use V3 if it's still in development? Are you saying to write them and ask to be a beta tester?

**Comment by oakwoodgates** [comment]
idk I'm assuming this much of it works http://help.convertkit.com/article/33-api-documentation-v3

**Comment by maelamb** [comment]
Ok @oakwoodgates and @earthwitch: Specifics 1-6 (with CK ID numbers) are in the Gdrive doc. All but the fourth one (badges) is outlined (I will do that one tomorrow AM).

**Comment by maelamb** [comment]
@earthwitch and @oakwoodgates: Specific No. 4 (the badges) is now also ready in the Gdrive doc. Thanks for your patience! :)
